packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -115,6 +115,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     fontSize: theme.typography.body1.fontSize,
   },
   feedName: {
+    fontSize: theme.typography.body2.fontSize,
```

Comment:
Also reverted this font size to a slightly smaller one, based on [this slack thread](https://cea-core.slack.com/archives/CJY3ZKP62/p1687398875474129)